### ISSUE TITLE
Adding repositories doen't retrieve contributions

### ISSUE THREAD

**Issue by Xoeseko** [sdgsolutionspace/sdg-market-frontend#3]
Just realized that adding repositories does not retrieve the desired contributions. I don't know what may be causing that since I can see my own contributions by opening them manually with the github API. 
For a second I thought it might be the slash in the url since the github API also returns an error for such a syntax but even when I removed it I still get an error while the github API retrieves the results. 
I then tried without punctuation in the repo name just in case but the error is still the same. 
Maybe the github API query rate limit was passed ?
If that is indeed the problem maybe log which administrator added which repo and have the requests to github made through this administrators login token rather close to how travis ci handles its queries... It would just require that every administrator generate a token for sdg-market. This multiplies the number of queries we could make since unsigned queries are limitted to 10 per minute while signed queries are limitted to 30 per minute per user !

tl;dr : Essentially no contributions are recorded for any newly added repo

**Comment by olivierb2** [comment]
I manually ran the cron on the testing server and check that cron service is properly running. Let me know if you have any other issue during the testing.

**Comment by Xoeseko** [comment]
Thanks for the quick response... I just checked the web app again and tokens for the last 3 projects added haven't evolved one single bit...

Also, when I retrieve contributions from the API I can only find contributions up to october 25th while running a similar query through the github api or simply looking manually at the repositories shows different results. 

Perhaps try checking anyone of the documentation repos... You should be able to see the difference between contributions retrieved by the web app and actual contributions to the repo.

**Comment by Xoeseko** [comment]
This is probably related to the previous issue but the api cannot retrieve my user information anymore, nor can I log in nor are any auctions displayed on the auctions page...
<img width="960" alt="api bug" src="https://user-images.githubusercontent.com/16857411/49647474-e7d30b00-fa22-11e8-99fe-5903e80600c4.PNG">
Just so you know

**Comment by gilles-hemmerle** [comment]
Are you still unable to login in the sdg-market pre prod server ? If you still note something not working properly on this server, please assign this ticket to Olivier. If not, please close this ticket.
